docs/heroku.md

```diff
@@ -0,0 +1,23 @@
+# Deploying to Heroku
+
+In order to deploy to heroku, you'll need to run this command once to set a custom
+buildpack:
+
+```
+heroku config:add BUILDPACK_URL=https://github.com/ddollar/heroku-buildpack-multi.git
+```
+
+This runs the two buildpacks in the `.buildpacks` directory.
+
+Also make sure you are running the latest heroku stack, cedar-14, to avoid running
+into the [following issue](https://github.com/sass/node-sass/issues/467#issuecomment-61729195).
+
+```
+heroku stack:set cedar-14 -a react-webpack-rails-tutorial
```

Comment:
Shouldn't `cedar-14` is the default by the way? should we mention `-a react-webpack-rails-tutorial` replace with their app name?